app/src/main/java/com/shagalalab/constitution/part/PartViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.shagalalab.constitution.part
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.shagalalab.constitution.data.dao.ChapterDao
+import com.shagalalab.constitution.data.dao.PartDao
+import com.shagalalab.constitution.data.models.PartModel
+import java.util.concurrent.Executors
+
+class PartViewModel(
+    private val partDao: PartDao,
+    private val chapterDao: ChapterDao
+) : ViewModel() {
+    private val partListLiveData: MutableLiveData<List<PartModel>> = MutableLiveData()
+    var partList: LiveData<List<PartModel>> = partListLiveData
+    var modelLiveData: MutableLiveData<Pair<Int, Boolean>> = MutableLiveData()
```

Comment:
Not a good naming, probably `chapterClickResultLiveData` might be better option. Make this private, then create its `LiveData` companion to observe from outside.